docs/GettingStarted/01-Why-Fedimint.md

```diff
@@ -1,75 +1,78 @@
 ---
 sidebar_position: 1
 ---
+
 # Why Fedimint
 
 :::caution
-This Guide is being actively worked on!
+This Guide is a work in progress! We'd appreciate any feedback, and you can submit edits through the link at the bottom of the page.
 :::
 
 Bitcoin is a powerful human rights technology that enables anybody in the world to **be their own bank**.
 
-The reality is that this requires a level of technical understanding and comfort that only comes with time and effort.  
+Anyone, anywhere in the world, can run their own node, custody their own funds, and transact permissionlessly over the Bitcoin main chain or the Lightning Network.
+
+We believe that
 
 Creating simpler, private user experiences will be critical in promoting the human rights benefits of bitcoin.
 
-Fedimint achieves this by focusing on three pillars.
+Fedimint is built on three guiding pillars.
 
 ![The spectrum and trade offs for Fedimint Custody](/img/raw-figures/fm-benefits.excalidraw.png)
 
 ## Community Custody
 
-The most popular convenient solution for new bitcoin users has been to purchase and store bitcoin on centralized exchanges.  
+Bitcoiners should run their own nodes and custody their own funds. However, the vast majority of people onboarding to Bitcoin today do so through 3rd party custodians like exchanges or custodial wallets. They sacrifice their privacy and security in favor of speed and convenience. We believe this tradeoff is unacceptable, and with Fedimint are building a solution which allows users to onboard to Bitcoin in a manner they find extremely convenient, without sacrificing privacy and security.
 
-Whist providing a good user experience this sacrifices key properties of bitcoin (private open banking) and represents a systemic risk to the bitcoin network of aggregating too many bitcoin in one custodian. 
```

Comment:
I think i want to keep the piece about systemic risk of aggregating into a custodian here, but the clean way to do it is probably to merge and then add this later :)